codec_specs.md

```diff
@@ -23,6 +23,7 @@ Video      | "V_"
 Audio      | "A_"
 Subtitle   | "S_"
 Button     | "B_"
+Timecode   | "T_"
```

Comment:
There is currently no "timecode" track type. There is a "metadata" track type which seems to be fitting. So maybe the prefix should be `"M_"` and the type called `Metadata` ?

We can also add a "timecode" track type. In the end it's more informational for the player what this is about, although it should be able to tell what kind of metadata we're talking about just by looking at the codec. But at the container level (don't know what's in codecs) it might be usefull to tell if we have timecodes or not.